cider/tests/utils/ArrowArrayBuilder.h

```diff
@@ -43,7 +43,7 @@ class ArrowArrayBuilder {
 
     array_->buffers = (const void**)allocator_->allocate(sizeof(void*));
     array_->buffers[0] = nullptr;
-    array_->n_buffers = 0;
+    array_->n_buffers = 1;
```

Comment:
Q: why do we need to set to 1 considering ```array_->buffers[0] = nullptr``` in L45?

Reply:
This buffer is allocated as a bitmap to show which children is NULL or not in **root** node of Arrow format. But it has not been used so far, so we temporarily set it to nullptr, and we should consider the number of buffers is 1. Otherwise, the later check in `StructBatch` will throw an exception.